packages/fx-core/templates/plugins/resource/bot/bicep/parameters.json

```diff
@@ -1,4 +1,4 @@
 {
-  "bot_aadClientId": "{{state.fx-resource-bot.botId}}",
-  "bot_aadClientSecret": "{{state.fx-resource-bot.botPassword}}"
+  "botAadAppClientId": "{{state.fx-resource-bot.botId}}",
```

Comment:
you may need to update migration logic either

Reply:
working on this.